tests/test_gates.py

```diff
@@ -122,50 +122,3 @@ def output(input):
     unitary_expected = qml.matrix(op(*p, wires=range(wires)))
 
     assert np.allclose(unitary, unitary_expected)
```

Comment:
We should also add tests that validate CPU state creation & validation, CPU state creation and validation on the GPU, and if possible, GPU state creation and validation.

Reply:
Add those tests in `test_initstate.py`.